service/history/historyEngineInterfaces.go

```diff
@@ -113,6 +113,7 @@ type (
 	transferQueueProcessor interface {
 		common.Daemon
 		NotifyNewTask(clusterName string, currentTime time.Time)
+		RetryTask(clusterName string)
```

Comment:
Why do we need another API for RetryTask?  Why Notify is not enough?

Reply:
notify, internally have a check to intentionally drop of the notification if the delay is not met. (for transfer queue side)
retry is just unblocking the standby tasks if there are any. those are total different.